app/services/reporting/total_user_count_report.rb

```diff
@@ -28,5 +28,21 @@ def total_user_count_emailable_report
     def total_user_count
       User.where('created_at <= ?', report_date).count
     end
+
+    def verified_user_count
+      Profile.where(active: true).where('activated_at <= ?', report_date).count
+    end
+
+    def annual_total_user_count
+      User.where(created_at: annual_start_date..end_date).count
+    end
+
+    def annual_start_date
+      (report_date - 1.year).beginning_of_day
+    end
+
+    def end_date
+      report_date.beginning_of_day
+    end
```

Comment:
I worked with @olatifflexion to ~steal his code~ incorporate his work on LG-11150 into this, rather than having him deal with merge conflicts.

We confirmed with Cody in the original doc that "annual" here refers to the previous 12 months, not fiscal/calendar year.